the-telephone-app-fe/src/components/Draw/Draw.js

```diff
@@ -232,55 +227,50 @@ const Draw = ({ width = "815rem", height = "350rem" }) => {
                 <div className="row mt-3 mb-3">
                   <div className="col-5 width-change-area card">
                     <div className="row">
-                      {widths.map((width) =>
-                        width === lineWidth ? (
-                          <div key={width} className="col-2">
-                            <div
-                              style={{ borderColor: drawColor }}
-                              onClick={() => changeLineWidth(width)}
-                              className="chosen-width-button m-3 ms-2 position-relative"
-                            >
-                              <div
-                                style={{
-                                  width: width + 2,
-                                  height: width + 2,
-                                  backgroundColor: drawColor,
-                                }}
-                                className="width-button-size position-absolute top-50 start-50 translate-middle"
-                              ></div>
-                            </div>
-                          </div>
-                        ) : (
-                          <div key={width} className="col-2">
+                      {widths.map((width) => (
+                        <div key={width} className="col-2">
+                          <div
+                            style={
+                              width === lineWidth
+                                ? { borderColor: drawColor }
+                                : {}
+                            }
+                            onClick={() => changeLineWidth(width)}
+                            className={
+                              width === lineWidth
+                                ? "chosen-width-button m-3 ms-2 position-relative"
+                                : "width-button m-3 ms-2 position-relative"
```

Comment:
use this syntax to avoid repeat other class
<div className={`${width === lineWidth ? ' chosen-width-button' : 'width-button'} m-3 ms-2 position-relative`}>
Please fix this in other places as well